### PR TITLE
Add a possibility to configure session_length_in_blocks

### PR DESCRIPTION
**Pull request by andor0**

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -2,11 +2,13 @@
 relaychain:
   image: parity/polkadot:v0.9.16 # the docker image to use
   chain: rococo-local # the chain to use
-  runtimeGenesisConfig: # additonal genesis override
-    configuration:
-      config:
-        validation_upgrade_cooldown: 10
-        validation_upgrade_delay: 10
+  runtime:
```
Comment (xlc) [inline config.yml:5]: is this breaking change necessary?

File: src/index.ts [hunk 2]
```diff
@@ -214,17 +217,20 @@ const generateRelaychainGenesisFile = (config: Config, path: string, output: str
...
       },
     ];
-    runtime.paras.paras.push(para);
+    (runtime.runtime_genesis_config || runtime).paras.paras.push(para);
   }
 
-  const tmpfile = `${shell.tempdir()}/${config.relaychain.chain}.json`;
+  // >>
+  // const tmpfile = `${shell.tempdir()}/${config.relaychain.chain}.json`;
+  const tmpfile = `/home/andor/work/composable/projects/centauri/tmp2/tmp/${config.relaychain.chain}.json`;
```
Comment (xlc) [inline src/index.ts:225]: leftover debug code?

**Review by xlc — CHANGES_REQUESTED** [review]

File: config.yml (hunk 1 above)
Comment (andor0) [reply config.yml:5]: I think not.

How about this?
```yml
relaychain:
  # ...
  runtimeGenesisConfig:
    configuration:
       config:
         validation_upgrade_cooldown: 10
         validation_upgrade_delay: 10
  runtimeConfig:
    session_length_in_blocks: 100
  env:
    # ...
```

Comment (xlc) [reply config.yml:5]: that will work. maybe we can just make it `overrides` or something similar to allow arbitrary chain spec overrides to make this more generic

Comment (andor0) [reply config.yml]: done.

**Review by xlc — APPROVED** [review]

**Pull request merged**